### PR TITLE
💄 Functions Rework Part 5 (general structure)

### PR DESCRIPTION
**Pull request by jameshughes89**
### What
Rework the general structure of a function 

### Additional Notes
Feels like the activities don't REALLY belong in the section, but whatever. They could go anywhere really in the latter half of the topic.

### CODE REVIEW

**Comment by twentylemon** [comment]
> 🆕 💣 🎓 📝 🤖 💄 🐛 🔭 🎨 Add relevant emojis to title. [See style guide.](https://github.com/jameshughes89/cs101/blob/main/CONTRIBUTING.md#style-guidelines).
> 
> ### Related Issues or PRs
> If applicable, replace this text with links to issues and other PRs. [Link with a keyword if you can.](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-issues/linking-a-pull-request-to-an-issue)
> 
> ### What
> Replace this text with a detailed description of what the change is.
> 
> ### Why
> Replace this text with a detailed description of why this is important.
> 
> ### Where
> Replace this text with a detailed description of where the change is and why (should it be an aside?).
> 
> ### How
> Replace this text with a detailed description of how it is addressed.
> 
> ### Testing
> If applicable, replace this text with a detailed description of how changes were tested. Include manual testing.
> 
> ### Additional Notes
> If applicable, replace this text with additional notes. Copyright concerns? etc.

File: site/topics/functions/functions.rst [hunk 1]
```diff
@@ -237,42 +237,54 @@ Abstraction
     * For example, when driving a car, do you think about the pistons firing?
 
 
-Back to concrete things...
-==========================
+Format of a Function
+====================
 
-* The general format for defining a function is::
+* The general format for defining a function is as follows
```
Comment (twentylemon) [inline site/topics/functions/functions.rst:243]: [Not quite](https://docs.python.org/3/reference/grammar.html). This is actually quite specific. Yeet that _general_.

File: site/topics/functions/functions.rst [hunk 2]
```diff
@@ -237,42 +237,54 @@ Abstraction
...
-* You tell Python which statements make up the *body* of the function by using *indentation*.
-    * This is a somewhat unique feature of Python. 
-        * And somewhat hated by some people. 
-    * Many other languages use pairs like ``begin, end`` , ``do, done`` or ``{, }`` to delimit the body of a function.
 
 .. admonition:: Activity
 
-    Write a function ``catstr`` which takes two strings as parameters and then prints out the concatenation of the strings. e.g., if I call ``catstr('Hello ','world!')`` it will print ``Hello world!``.
+    Write a function ``concatenate_strings`` that takes two strings as parameters and then returns the concatenation of
+    the strings. For example, if I call ``concatenate_strings("Hello","World!")`` it will return ``Helloworld!``. Do
```
Comment (twentylemon) [inline site/topics/functions/functions.rst:268]: Should be `HelloWorld!` -- capital `W`

**Review by twentylemon — APPROVED** [review]

File: site/topics/functions/functions.rst (hunk 2 above)
Comment (jameshughes89) [reply site/topics/functions/functions.rst:268]: 1. I originally had it that way
2. Changed it to emphasize that there is no space between them
3. I later added the thing saying "don't add a space"

In other words, I'll put it back --- good catch.

**Pull request merged**